UsbIpServer/Program.cs

```diff
@@ -26,6 +26,7 @@ namespace UsbIpServer
     static class Program
     {
         const string InstallWslUrl = "https://aka.ms/installwsl";
+        const string SetWslVersionUrl = "https://docs.microsoft.com/en-us/windows/wsl/basic-commands#set-wsl-version-to-1-or-2";
```

Comment:
If you change this to `https://docs.microsoft.com/windows/wsl/basic-commands#set-wsl-version-to-1-or-2` it should go to the properly localized page based on the user's locale settings.